tests/melt_track.prm

```diff
@@ -0,0 +1,200 @@
+# Listing of Parameters
+# ---------------------
+# Set up the solitary wave benchmark
+# (Barcilon & Richter, 1986; Simpson & Spiegelman, 2011; 
+# Keller et al., 2013; Schmeling, 200
```

Comment:
Please modify this comment, and add something about the new purpose of this test.